core/src/main/java/org/apache/carbondata/core/util/TrashUtil.java

```diff
@@ -192,12 +192,9 @@ public static void emptyTrash(String tablePath) {
   }
 
   /**
-   * This will tell whether the trash retention time has expired or not
-   *
-   * @param fileTimestamp
-   * @return
+   * whether trash data inside of .Trash folder is time out
```

Comment:
now we are using this for data outside trash also, so change the description.

`returns true, if the current timestamp has expired based on carbon.trash.retention.days `

Reply:
no